src/otx/core/model/entity/base.py

```diff
@@ -5,17 +5,19 @@
 
 from __future__ import annotations
 
+import json
 import warnings
 from abc import abstractmethod
 from typing import TYPE_CHECKING, Any, Generic, Iterable, Iterator, NamedTuple
 
+import nncf
```

Comment:
Is this needed? I don't want to import this at the top level because in my old experiece, importing `nncf` was heavy (it can be wrong).
```suggestion
```

Reply:
Ok. Will move it to optimize()